### DISCUSSION TITLE
Is there a simple way to populate a field with the server's current time?

### DISCUSSION THREAD

**Discussion by dtau00** [appy-one/acebase#157] [Q&A] [answered]
Perhaps AcebaseClient maintains a synced servertime internally, that i can use to populate last_updated fields locally?  It wouldn't be 100% accurate but good enough.  This would make it very simple to update last_updated via a live proxy.  I thought about a magic property value but that would have issues with the local cache.

**Comment by appy-one** [comment, marked as answer]
Great idea, I'll have a look at creating a `getServerTime` function

**Reply by appy-one** [reply]
Looking at the code, there should already be a way to do this...
Try this:
```js
import { ServerDate } from 'acebase-client';
// or: const { ServerDate } = require('acebase-client');
const serverDateTime = new ServerDate();
```

**Reply by dtau00** [reply]
sweet perfect!
